api/ws/server.go

```diff
@@ -33,41 +37,52 @@ var (
 	ErrExpired = errors.New("expired")
 )
 
-func WithWebsocketAPI(maxPendingMessages int) vm.Option {
-	return func(v *vm.VM) error {
-		actionRegistry, authRegistry := v.Registry()
-		server, handler := NewWebSocketServer(
-			v,
-			v.Logger(),
-			v.Tracer(),
-			actionRegistry,
-			authRegistry,
-			maxPendingMessages,
-		)
-
-		webSocketFactory := NewWebSocketServerFactory(handler)
-		txRemovedSubscription := subscriptionFuncFactory[vm.TxRemovedEvent]{
-			AcceptF: func(event vm.TxRemovedEvent) error {
-				return server.RemoveTx(event.TxID, event.Err)
-			},
-		}
+type Config struct {
+	MaxPendingMessages int `json:"maxPendingMessages"`
```

Comment:
Nice! Separate from this PR, we can add `Enabled` here as well, so that this can be easily disabled via config instead of via code